### ISSUE TITLE
4.7 requires Syncfusion license (may apply only to Git source)

### ISSUE THREAD

**Issue by peppersass** [tomhollander/PowerwallCompanion#12]
When 4.7 is fetched from GitHub, built, deployed and run, it displays the following pop-up when launched:

Syncfusion License

This application was built using a trial version of Syncfusion Essential Studio. To remove the license validation message permanently, a valid license key must be included.

You can hit OK to make the message go away, or it'll disappear after a few seconds.

It's just an annoyance, but can this be fixed?

**Comment by tomhollander** [comment]
You can sign up for your own API key at the SyncFusion website - keys are free for non-commercial apps. I don't include the key used in the store app in the Github rep for obvious reasons.

**Comment by peppersass** [comment]
Is the API key the same as the license key? I tried signing up for a free license key but they require a link to my LinkedIn or Xing profile. I don't use social media, so I don't have one. I used yours (hope you don't mind.) They're going to verify within 48 hours. Hope it works. Meanwhile, they sent me a trial key. Where does the key go in the project folder?

I guess if I can't get the key I'll install the store version, give up on modifying 4.7 and just use my modified version of 3.6 for daily scheduled downloads.
